CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,128 @@
+# Contributor Covenant Code of Conduct
+
+## Our Pledge
+
+We as members, contributors, and leaders pledge to make participation in our
+community a harassment-free experience for everyone, regardless of age, body
+size, visible or invisible disability, ethnicity, sex characteristics, gender
+identity and expression, level of experience, education, socio-economic status,
+nationality, personal appearance, race, religion, or sexual identity
+and orientation.
+
+We pledge to act and interact in ways that contribute to an open, welcoming,
+diverse, inclusive, and healthy community.
+
+## Our Standards
+
+Examples of behavior that contributes to a positive environment for our
+community include:
+
+* Demonstrating empathy and kindness toward other people
+* Being respectful of differing opinions, viewpoints, and experiences
+* Giving and gracefully accepting constructive feedback
+* Accepting responsibility and apologizing to those affected by our mistakes,
+  and learning from the experience
+* Focusing on what is best not just for us as individuals, but for the
+  overall community
+
+Examples of unacceptable behavior include:
+
+* The use of sexualized language or imagery, and sexual attention or
+  advances of any kind
+* Trolling, insulting or derogatory comments, and personal or political attacks
+* Public or private harassment
+* Publishing others' private information, such as a physical or email
+  address, without their explicit permission
+* Other conduct which could reasonably be considered inappropriate in a
+  professional setting
+
+## Enforcement Responsibilities
+
+Community leaders are responsible for clarifying and enforcing our standards of
```

Comment:
Who are the community leaders? @pzaino – are you up for this? As I wrote in my general comment, I think it is important not to just add a document like this, without a plan for it.

Reply:
I am ok to be in the list, I'd recommend to add others as well due to time constraints. Also technically the EU now requires/enforces this anyway and so does the US Gov, so even without publishing it we'd be subject to github.com policies anyway.

Reply:
> Also technically the EU now requires/enforces this anyway and so does the US Gov,

What does this refer to?

Reply:
So, here are some refs guys:

https://ec.europa.eu/commission/presscorner/detail/en/qanda_20_1135
https://www.europarl.europa.eu/RegData/etudes/STUD/2020/652718/IPOL_STU(2020)652718_EN.pdf

UK Gov at this time is still fully adhering to the EU regulations on the matter: 

https://www.gov.uk/government/publications/code-of-practice-for-providers-of-online-social-media-platforms/code-of-practice-for-providers-of-online-social-media-platforms

The long story short is: We should offer a clear code of conduct, so that every user would know what is the allowed behaviour. If we avoid that then such allowed behaviour is actually still in place, so people should follow it anyway, however in such a case I am not sure how the community could be affected in case of confirmed miss-behaviour of some if we did not follow the directives to inform all the users of what is the allowed behaviour. 

Am I making sense? Sorry if I am not, a) I am not a layer and b) not native English speaker, but the links above should be enough informative on all the details and responsibilities.